pkg/auth/serviceaccount_test.go

```diff
@@ -108,6 +110,9 @@ func TestGetOIDCServiceAccountForResource(t *testing.T) {
 			Annotations: map[string]string{
 				"description": "Service Account for OIDC Authentication for Broker \"my-broker\"",
 			},
+			Labels: map[string]string{
+				sources.OIDCLabelKey: "",
```

Comment:
same nit as above, would need to update here for the tests to pass if we make the change to have an actual value for this label